examples/example_rn/posthog.tsx

```diff
@@ -2,10 +2,11 @@ import PostHog, {PostHogProvider} from 'posthog-react-native';
 import React from 'react';
 
 export const posthog = new PostHog(
-  'phc_pQ70jJhZKHRvDIL5ruOErnPy6xiAiWCqlL4ayELj4X8',
+  'phc_QFbR1y41s5sxnNTZoyKG2NJo2RlsCIWkUfdpawgb40D',
```

Comment:
Are you happy to switch this api key?

Reply:
yeah its just the sample and the other one is rotated